src/pages/HomePage.jsx

```diff
@@ -0,0 +1,75 @@
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { IoMdMic } from 'react-icons/io';
+import { FiSettings } from 'react-icons/fi';
+import { AiOutlineLeft } from 'react-icons/ai';
+import { fetchCountriesData } from '../Redux/Countries/Countries';
+import './HomePage.scss';
+import { getMapUrl } from '../Utilities/Map';
+import CountriesList from '../components/CountriesList';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const { status, data } = useSelector((state) => state.Countries);
+  const [search, setSearch] = useState('');
+
+  useEffect(() => {
+    if (status === 'Not Fetched') {
+      dispatch(fetchCountriesData());
+    }
+  }, []);
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you pay attention to the console so that you can write more conventional React code. From the warning message and another one, you are missing `dispatch` and `status` in the dependency array.